GSA-TTS/FAC
# 3724 address historical audits with missing ueis

## Description
This PR addresses the issue related to invalid UEIs. Some historical records are either missing the auditee UEI, or contain an auditee UEI value that will fail validation against the application's UEI schema with the error message `'' is too short`. The team-approved solution is to replace the missing or invalid value with `GSA_MIGRATION`.
Changes include:
Update logic to replace invalid/missing UEI field with `GSA_MIGRATION` in the general information section of historical records.
Ensure the schema allows `GSA_MIGRATION` in UEI fields.
Update logic to only allow `GSA_MIGRATION` in UEI fields when the migration flag is present.

How to Test:
1. Launch the app from this PR branch and modify the value of the UEI field in `ELECAUDITHEADER` table for an existing record and attempt to migrate this report.
2. Observe that the auditee_uei field in the migrated record (in general table) contains `GSA_MIGRATION`


## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
